lib/awful/rules.lua

```diff
@@ -262,32 +318,26 @@ function rules.execute(c, props, callbacks)
         end
     end
 
-    for property, value in pairs(props) do
+    -- Before requesting a tag, make sure the screen is right
```

Comment:
I agree with this comment. Doesn't that mean that this code should be before the previous for-loop?